R/grade_learnr.R

```diff
@@ -198,3 +199,19 @@ learnr_env <- function(learnr_args) {
     learnr_args$envir_prep %||%
     stop("Internal error. learnr did not pass a relevant environment")
 }
+
+
+#' Enable gradethis in learnr
+#' 
+#' Call this function inside a learnr document to use gradethis' suggested
+#' setup for grading exercises.
+#' 
+#' @export
+#' @seealso [grade_learnr()]
+enable_gradethis <- function() {
```

Comment:
This function is a great idea, enable_gradethis

Reply:
@garrettgman Do you like the name? `gradethis_learnr()`?

Reply:
Oooo what about `gradethis_setup()`